<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentDetailWidget</class>
 <widget class="QWidget" name="PaymentDetailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>506</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="PaymentDetailTitle" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>37</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PaymentDetailView" name="PayDetailsView">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="pageLabel">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>1/1é¡µ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="upPageButton">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>87</width>
          <height>37</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border:none;
	border-image: url(:/saleImage/btn-up-nomal.png);
}

QPushButton:pressed
{
	border-image: url(:/saleImage/btn-up-pess.png);
}

QPushButton:disabled
{
	border-image: url(:/saleImage/btn-up-disable.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#frame {
	border:1px solid  rgb(237, 237, 237);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downPageButton">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>87</width>
          <height>37</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border:none;
	border-image: url(:/saleImage/btn-down-nomal.png);
}

QPushButton:pressed
{
	border-image: url(:/saleImage/btn-down-pess.png);
}

QPushButton:disabled
{
	border-image: url(:/saleImage/btn-down-disabled.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaymentDetailTitle</class>
   <extends>QWidget</extends>
   <header>PaymentDetailTitle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PaymentDetailView</class>
   <extends>QListView</extends>
   <header>PaymentDetailView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
